.gitignore

```diff
@@ -1,22 +0,0 @@
-.antlr*
```

Comment:
You rather not delete this

Reply:
Just for my information can you please tell me what is the purpose of this .antlr line?

Reply:
This was indeed the wrong .gitignore to be deleted. I was referring to the gitignore in the siemensrds binding: https://github.com/openhab/openhab2-addons/blob/master/bundles/org.openhab.binding.siemensrds/.gitignore

.antlr are generated ANTLR generator tool generated files that are generated at compile time and therefor should not be put into git. I don't know why it's here. The only thing I can think of is it's copied from the core project that uses xtend that uses antlr inside.

Reply:
Ok, confirmed deleted this.